pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -143,7 +143,7 @@ func (r *remoteRuntimeService) determineAPIVersion(conn *grpc.ClientConn) error
 		r.runtimeClientV1alpha2 = runtimeapiV1alpha2.NewRuntimeServiceClient(conn)
 
 	} else {
-		return fmt.Errorf("unable to determine runtime API version: %w", err)
+		return fmt.Errorf("unable to determine image API version with %q, or API is not implemented: %w", endpoint, err)
```

Comment:
why change from `runtime` to `image`?

Reply:
+1 I think it is a copy/paste error =)

```suggestion
		return fmt.Errorf("unable to determine runtime API version with %q, or API is not implemented: %w", endpoint, err)
```

Reply:
Ah yes, it is a paste error, updated.
PTAL thanks.